Makefile

```diff
@@ -72,8 +80,10 @@ serve: html
 
 
 clean: clean-doc
-	@test ! -e "$(VENVDIR)" -o -d "$(VENVDIR)" -a "$(abspath $(VENVDIR))" != "$(VENVDIR)"
+	@test ! -e "$(VENVDIR)" -o -d "$(VENVDIR)" \
+        -a "$(abspath $(VENVDIR))" != "$(VENVDIR)"
 	rm -rf $(VENVDIR)
+	@{ echo "Removing \"woke\" snap... \n"; sudo snap remove woke; }
```

Comment:
No, we can't do that - I have woke already installed on my system, clean shouldn't remove it.

There's probably no way to install woke locally somehow in case it's not installed globally, or is there? That would be cleaner, and then it could of course also be removed afterwards.

Reply:
In general yes - but we can't reconfigure snapd and reboot the machine. :(

Reply:
Maybe @degville has some suggestion here?

Reply:
Sure - the other changes all look good to me. :)